assets/css/components/_brand.scss

```diff
@@ -0,0 +1,39 @@
+.logo {
+  width: calc(200 / 16 * 1rem);
+}
+
+[class^="fl-theme"] .logo path,
+[class^="fl-theme"] .logo circle,
+[class^="gpii-ext-theme"] .logo path,
+[class^="gpii-ext-theme"] .logo circle {
+  fill: none;
+  stroke: var(--logo-stroke);
```

Comment:
@greatislander you can simplify this by just setting stroke to [`currentColor`](https://css-tricks.com/currentcolor/). This will just pick up the current color value which is set through the contrast change. This will eleminate the need to set the color variables, have theme specific styling declarations, and should be more flexible for handling future changes to the contrast themes.

Reply:
Because the SVG image is within a link, using `currentColor` adopts the link colour for the dark grey, light grey, and black on brown colour schemes. Since it works properly for the others, I'm wondering if this is an issue with UIO+?

Reply:
Ah I see, well that's not an issue with UIO+ that's because those contrast themes aren't binary and set different colours for links. So, yes in those cases then the colour would be different. 

It's an interesting problem because on one hand the contrast theme is saying that links should be a different colour. On the other hand, I'm not sure you want to treat the site logo the same as the text links on the page.

Curious about your thoughts. Below are a few options I can think of:

- modify the themes so that they don't style SVGs in the same way as other links, at least in terms of colour. Although this would affect other links.
- provide a class or some other mechanism to target certain elements to not style like the rest of the links.
- leave this to the integrator to handle ( i.e. you would have to override them in your styles in this case)

Reply:
My inclination would be to specify the SVG colour (grey for light/dark grey theme and black for black on brown theme) within the stylesheet here. Thoughts? I agree, using the link colour doesn't feel quite right.

Reply:
That would be the least prescriptive approach in terms of modifying UIO themes. I guess we can revisit the issue in UIO if we run into this a lot and it because annoying or error prone to handle.

Reply:
Sounds good. One clarification: UIO+ light grey/dark grey body classes are `gpii-ext-theme-gw` and `gpii-ext-theme-gd` respectively. In the UIO [integration guide](https://docs.fluidproject.org/infusion/development/tutorial-userInterfaceOptions/UsingImagesInContrastModes.html), it looks like the equivalent to the dark grey class is `fl-theme-lgdg`. Has anything changed here that I should be aware of on the UIO side?

Reply:
This is the [styling](https://github.com/GPII/gpii-chrome-extension/blob/master/extension/stylus/enactors.styl#L13-L60) for the contrast themes used by UIO+. 

This is the [styling](https://github.com/fluid-project/infusion/blob/master/src/framework/preferences/css/stylus/utils/Themes.styl#L4-L55) for the contrast themes used by UIO.

The prefs framework/UIO supports an additional theme light-grey-dark-grey (lgdg) which we don't use in UIO+, but you'll probably want to support with the integration of UIO on the site. It won't be affected by the link colour issue we are talking about here though, because it is also a binary theme.

Reply:
Seems like the integration guide is a bit out-of-date :(